hermes-consumers/src/test/java/pl/allegro/tech/hermes/consumers/consumer/ConsumerMessageSenderTest.java

```diff
@@ -274,6 +276,38 @@ public void shouldBackoffRetriesWhenEndpointFails() throws InterruptedException
         verifyErrorHandlerHandleFailed(message, subscriptionWithBackoff, 1 + executionTime / senderBackoffTime);
     }
 
+    @Test
+    public void shouldBackoffRetriesOnRetryAfter() throws InterruptedException {
+        // given
+        int retrySeconds = 1;
```

Comment:
I was not aware of that. Weird choice. We could add custom header Retry-After-Millis and accept milliseconds but we have to test original Retry-After anyway so your point is valid.